apps/src/aiDifferentiation/welcome/ai-diff-welcome.module.scss

```diff
@@ -232,6 +232,8 @@
   align-items: center;
   margin: 0 36px 16px 36px;
   text-align: center;
+  overflow: hidden;
+  padding: 0 4px;
```

Comment:
I needed to add padding right and left to keep the borders of the `ai101` image.